sql/plan/ddl.go

```diff
@@ -50,6 +59,12 @@ func (c *CreateTable) Resolved() bool {
 
 // RowIter implements the Node interface.
 func (c *CreateTable) RowIter(s *sql.Context) (sql.RowIter, error) {
+	creatable, ok := c.db.(sql.TableCreator)
+	if ok {
+		return sql.RowsToRowIter(), creatable.CreateTable(s, c.name, c.schema)
```

Comment:
I had to test it myself but I guess it does return `0 rows affected` on success, which makes sense since literally no rows were affected. Seems weird to me, like it should say `1 table created` or something.